src/test/java/Tests.java

```diff
@@ -0,0 +1,98 @@
+import org.junit.Ignore;
+import org.testng.annotations.Test;
+import parser.JsonParser;
+import parser.NoSuchFileException;
+import shop.Cart;
+import shop.RealItem;
+import shop.VirtualItem;
+import java.io.File;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import static org.testng.AssertJUnit.*;
+import static org.junit.jupiter.api.Assertions.assertAll;
+
+public class Tests {
+    @Test
+    public void TestJSonParserWriteToFile(){
```

Comment:
java naming convention: method name should be specified in camelCase notation. for this exact test it would be like `testJSonParserWriteToFile `